.github/workflows/run-performance-tests-reusable.yaml

```diff
@@ -0,0 +1,329 @@
+name: Run performancestress tests (reusable)
+
+on:
+  workflow_call:
+    inputs:
+      image-repo:
+        description: Binary image registry reference
+        required: true
+        type: string
+      image-tag:
+        description: Binary image tag
+        required: true
+        type: string
+      credentials-mode:
+        description: real or dummy credentials
+        required: true
+        type: string
+
+jobs:
+  frequent-secret-update-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Set up go environment
+        uses: actions/setup-go@v5
+        with:
+          go-version-file: 'go.mod'
+
+      - name: Prepare k3s cluster and docker registry
+        run: "./scripts/testing/k3s-setup.sh --wait"
+
+      - name: Wait for artifacts
+        env:
+          IMAGE_REPO: ${{ inputs.image-repo }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: "./scripts/await_image.sh ${{ inputs.image-tag }}"
+
+      - name: Install BTP Manager and SAP BTP Operator
+        timeout-minutes: 2
+        run: "./scripts/testing/install_module.sh ${{ inputs.image-repo }}:${{ inputs.image-tag }} ${{ inputs.credentials-mode }} ci"
+
+      - name: Edit sap-btp-manager secret in a loop
+        run: |
+            for i in {1..10}; do
+              kubectl -n kyma-system patch secret sap-btp-manager --type='json' -p='[{"op": "replace", "path": "/data/sm_url", "value": "bmV3X3NtX3VybA=="}]'
```

Comment:
Could we use new feature allowing to set value not base64 encoded? Does it make sense?

Reply:
This is irrelevant. I wanted only to trigger the reconcile by changing something.